cfme/tests/physical_infrastructure/ui/test_physical_switch_list.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
```

Comment:
**REQUIRED:** I don't think we need a fixture for the collection. It can just be directly accessed in the tests.

Reply:
@psav Could you confirm if @matheuscmelo's comment is what you suggest?

Reply:
Yes that's what I meant

Reply:
@matheuscmelo @psav answered yes on gitter. I am doing the change.